### PR TITLE
 adding the vuevalidate

### PR DESCRIPTION
**Pull request by MouaadDilmi**
adding vuelidate of password and email

### CODE REVIEW

File: frontend/src/views/Auth.vue [hunk 1]
```diff
@@ -102,44 +160,53 @@ export default {
...
-        await this.$router.push({ name: 'Home' });
+        this.$router.push({ name: 'Home' });
       } catch (error) {
         this.error = 'Invalid email or password.';
       }
     },
     async signUp() {
       try {
         await auth.createUserWithEmailAndPassword(this.email, this.password);
+        await this.bindProfiles();
```
Comment (DorianWilhelm) [inline frontend/src/views/Auth.vue]: Where does this come from?

File: frontend/src/views/Auth.vue [hunk 2]
```diff
@@ -102,44 +160,53 @@ export default {
...
+      this.$v.$touch();
+      if (this.$v.$invalid) {
+        this.submitStatus = 'ERROR';
+        return;
       }
+      this.submitStatus = 'PENDING';
+
+      if (this.mode == 'signin') {
+        this.signIn();
+      }
```
Comment (DorianWilhelm) [inline frontend/src/views/Auth.vue:149]: here is an else missing, b/c signup is called everytime

File: frontend/src/views/Auth.vue [hunk 3]
```diff
@@ -102,44 +160,53 @@ export default {
...
+      this.signUp();
     },
     async signIn() {
       try {
         await auth.signInWithEmailAndPassword(this.email, this.password);
+        await this.bindProfiles();
```
Comment (DorianWilhelm) [inline frontend/src/views/Auth.vue]: where does this come from?

**Review by DorianWilhelm — CHANGES_REQUESTED** [review]

**Comment by DorianWilhelm** [comment]
Signin and signup is not working!

**Comment by gabrielheinrich** [comment]
See new pull request

**Pull request closed without merging**